plugin/src/main/java/at/helpch/chatchat/config/holder/MessagesHolder.java

```diff
@@ -36,6 +37,8 @@ public final class MessagesHolder {
     private Component unignoredPlayer = text("Successfully un-ignored <player>.", GREEN);
     private Component alreadyIgnored = text("You are already ignoring <player>.", RED);
     private Component notIgnored = text("You are not ignoring <player>.", RED);
+    private Component notIgnoringAnyone = text("You are not ignoring anyone.", RED);
+    private Component ignoredPlayersList = text("You are ignoring following players: <white><ignored_players>.", YELLOW);
```

Comment:
This isn't exactly what I meant when I said you can append components. I'm pretty sure this will not actually work as you intended it to. This is what I meant:
```java
private Component ignoredPlayersList = text("You are ignoring following players:", YELLOW)
        .append(text("<ignored_players>", WHITE))
        .append(text(".", YELLOW));
```